.github/workflows/tests.yml

```diff
@@ -20,25 +29,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-          python-version: [3.6, 3.7, 3.8]
-          # You can add more combinations e.g. dev requirements or MySQL by adding
-          # a new item to this list. Add its corresponding definition below.
-          services: [release, lowest, devel]
-          include:
-          - services: release
-            REQUIREMENTS: "release"
-            REQUIREMENTS_LEVEL: "pypi"
-          - services: lowest
-            REQUIREMENTS: "lowest"
-            REQUIREMENTS_LEVEL: "min"
-          - services: devel
-            REQUIREMENTS: "devel"
-            REQUIREMENTS_LEVEL: "dev"
```

Comment:
I am not sure if you were planning to remove the devel and lowest, it could make sense to leave them, @lnielsen what do you think?

Reply:
I would keep ``min`` (changing name from lowest), but remove ``devel``.